cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -12,124 +12,249 @@ import "protoc-gen-openapiv2/options/annotations.proto";
-    option (google.api.http) = {
-      get: "/core/packages/v1alpha1/packagemeta"
-    };
-  }
+
+  // TODO: remaining operations yet to be defined
+
+  // rpc GetAvailablePackageVersions(GetAvailablePackageVersionsRequest) returns (GetAvailablePackageVersionsResponse) {
+  //   option (google.api.http) = {
+  //     get: "/core/packages/v1alpha1/availablepackageversions"
+  //   };
+  // }
+  // rpc GetInstalledPackageSummaries(GetInstalledPackageSummariesRequest) returns (GetInstalledPackageSummariesResponse) {
+  //   option (google.api.http) = {
+  //     get: "/core/packages/v1alpha1/installedpackagesummaries"
+  //   };
+  // }
+  // rpc GetInstalledPackageDetail(GetInstalledPackageDetailRequest) returns (GetInstalledPackageDetailResponse) {
+  //   option (google.api.http) = {
+  //     get: "/core/packages/v1alpha1/installedpackagedetails"
+  //   };
+  // }
+  // rpc CreateInstalledPackage(CreateInstalledPackageRequest) returns (CreateInstalledPackageResponse) {
+  //   option (google.api.http) = {
+  //     post: "/core/packages/v1alpha1/installedpackages"
+  //   };
+  // }
+  // rpc UpdateInstalledPackage(UpdateInstalledPackageRequest) returns (UpdateInstalledPackageResponse) {
+  //   option (google.api.http) = {
+  //     put: "/core/packages/v1alpha1/installedpackages"
+  //   };
+  // }
+  // rpc DeleteInstalledPackage(DeleteInstalledPackageRequest) returns (DeleteInstalledPackageResponse) {
+  //   option (google.api.http) = {
+  //     delete: "/core/packages/v1alpha1/installedpackages"
+  //   };
+  // }
 }
-//
+
 // Standard request and response messages for each required function are defined
 // below for import and re-use.
 
-// GetAvailablePackages
+// -- Start definitions of the request messages -- 
+
+// GetAvailablePackageSummariesRequest
 //
-// Request for GetAvailablePackages
-message GetAvailablePackagesRequest {
-  // A cluster name can be provided if multiple clusters are configured,
-  // otherwise the current cluster will be assumed.
-  string cluster = 1;
+// Request for GetAvailablePackageSummaries
+message GetAvailablePackageSummariesRequest {
+  // The context (cluster/namespace) for the request
+  Context context = 1;
+  // TODO: Add standard filters.
+}
 
-  // A namespace can be provided if the packages available for install in a
-  // specific namespace are required.
-  string namespace = 2;
+// GetAvailablePackageDetailRequest
+//
+// Request for GetAvailablePackageDetail
+message GetAvailablePackageDetailRequest {
+  // The information required to uniquely
+  // identify an available package
+  AvailablePackageReference available_package_ref = 1;
 
+  // Optional specific version (or version reference) to request.
+  // By default the latest version (or latest version matching the reference)
+  // will be returned.
+  string version = 2;
+}
+
+// GetPackageRepositories
+//
+// Request for GetPackageRepositories
+message GetPackageRepositoriesRequest {
```

Comment:
I'm not sure if we actually want/need any CRUD API for repositories right now? We implemented this to check a few things, but I'm wanting to focus solely on the packaging API for now.

Reply:
Yep, delete this and the related response.

Reply:
Just to confirm: you mean removing every reference to `GetPackageRepositories` (in both proto and go code), do you?